CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 ## 6.2.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_certificate_manager_certificates` ([#19361](https://github.com/hashicorp/terraform-provider-google/pull/19361))
+* **New Resource:** `google_network_security_server_tls_policy` ([#19314](https://github.com/hashicorp/terraform-provider-google/pull/19314))
+* **New Resource:** `google_scc_v2_folder_scc_big_query_exports` ([#19327](https://github.com/hashicorp/terraform-provider-google/pull/19327))
+* **New Resource:** `google_scc_v2_project_scc_big_query_exports` ([#19311](https://github.com/hashicorp/terraform-provider-google/pull/19311))
```

Comment:
Shouldn't these resources have been named `google_scc_v2_folder_scc_big_query_export` and `google_scc_v2_project_scc_big_query_export` instead? Here's a [link to the API docs](https://cloud.google.com/security-command-center/docs/reference/rest/v2/projects.locations.bigQueryExports/get) where a singular resource is referred to as an export.

Sorry to bring this up in a CHANGELOG review, but I think this is a mistake we have an opportunity to correct

Reply:
+1 good catch!

Reply:
they've been renamed, cherrypicked, and this has been updated